.github/workflows/publish-storybooks.yml

```diff
@@ -34,12 +37,14 @@ jobs:
           yarn install --frozen-lockfile
           yarn build
           yarn build-storybook
-          echo ::set-output name=success::$(echo "true")
-        #          echo "style.monday.com" > ./CNAME
         env:
-          NODE_AUTH_TOKEN: ${{ secrets.npm_token }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: Set up git credentials
+        run: |
+          git config --global user.name 'ci'
+          git config --global user.email 'deploy@monday.com'
       - name: "Build failed - exit"
-        if: ${{ steps.build_step.outputs.success != 'true' }}
+        if: success() != true
```

Comment:
Does it mean it will run everytime? On every merge to master?

Reply:
Was in this PR by mistake - removed it from here already, And it's merged already also, cause I needed to test it out from master
Fixed it there to run after release is completed https://github.com/mondaycom/vibe-lerna-test/pull/27